packages/playwright-core/src/server/bidi/third_party/bidiCommands.d.ts

```diff
@@ -118,6 +118,11 @@ export interface Commands {
     returnType: Bidi.EmptyResult;
   };
 
+  'emulation.setLocaleOverride': {
```

Comment:
I don't see this command in https://github.com/puppeteer/puppeteer/blob/main/packages/puppeteer-core/src/bidi/core/Connection.ts which this file is copied from (see the comment at the top). Maybe we can update the upstream file and just roll it to avoid merge conflicts in the future? Chrome will need the same command anyway.

Reply:
I was under the impression that these files aren't being synced since this file also contains `network.setCacheBehavior` which isn't present upstream.